graphman-client/modules/graphman.js

```diff
@@ -83,7 +94,7 @@ module.exports = {
                 } else {
                     utils.info("unexpected graphman http response");
                     utils.info(data);
-                    callback({error: data, data: {}});
+                    callback({ error: data, data: {} });
```

Comment:
please avoid **indentation-only** changes.
You may certainly improve the indention of the changed code.